paddle/phi/kernels/onednn/softplus_kernel.cc

```diff
@@ -29,11 +29,10 @@ class SoftplusOneDNNHandler
       : funcs::OneDNNHandlerNoCachingT<T, dnnl::binary>(dev_ctx.GetEngine(),
                                                         dev_ctx.GetPlace()) {
     dnnl::post_ops post_ops;
-    post_ops.append_eltwise(
-        1.0f, dnnl::algorithm::eltwise_soft_relu, 0.0f, 0.0f);
+    post_ops.append_eltwise(dnnl::algorithm::eltwise_soft_relu, 0.0f, 0.0f);
```

Comment:
Just a reminder:
In oneDNN v3 API, the formula of soft_relu is changed, so we may need to set the `alpha` to `1.f`
- v2.7 soft_relu: $d = \log_{e}(1+e^s)$ https://oneapi-src.github.io/oneDNN/v2.7/dev_guide_eltwise.html
- v3 soft_relu: $d =\frac{1}{\alpha} \log_{e}(1+e^{\alpha s})$ https://oneapi-src.github.io/oneDNN/dev_guide_eltwise.html

Reply:
@YangQun1 Thanks, will remember to modify this